blablas/ep173/index.md

```diff
@@ -0,0 +1,54 @@
+---
+date: 2024-03-10
+time: 21h:00min
+duration: "1:29:29"
+title: "#173 - Blanat Coding Challenge #BCC discussion"
+tags: ["DEV"]
+category: "DEV"
+isNext: false
+youtube: https://www.youtube.com/watch?v=gOnapDngTuk
+published: true
+featured: false
+---
+
+In this episode, we discuss how to prepare for talks, taking 3 months long vacations and the future of the tech industry in 2024.
```

Comment:
@adnanmerrakchi  apparently you missed updating the description

Reply:
done